src/css-loader-core/loader.js

```diff
@@ -44,6 +44,14 @@ const traceKeySorter = (a, b) => {
 
 export default class FileSystemLoader {
   constructor(root, plugins) {
+    if (root === '/' && process.platform === "win32") {
+      const cwdDrive = process.cwd().slice(0, 3)
+      if (!/^[A-Z]:\\$/.test(cwdDrive)) {
+        throw new Error(`Failed to obtain root from "${process.cwd()}".`)
+      }
+      root = cwdDrive
+    }
```

Comment:
I added these line here instead of `getLoader` to avoid a breaking change.
https://github.com/madyankin/postcss-modules/blob/7d5965d4df201ef301421a5e35805d1b47f3c914/src/index.js#L34